### ISSUE TITLE
When idleStart is called don't toggleInterrupts(false)

### ISSUE THREAD

**Issue by paullryan** [HackedByChinese/ng2-idle#5]
Currently in Idle.prototype.toggleState when idleStart triggers it triggers the this.toggleInterrupts(false) which never allows idleEnd to occur. Seems like there needs to be one more condition here of not just this.idling but also the parent that called toggleState.

**Comment by paullryan** [comment]
Seems related to #4

**Comment by paullryan** [comment]
Maybe something like

```js
    Idle.prototype.timeout = function () {
        this.stopKeepalive();
        this.toggleInterrupts(false);
        this.safeClearInterval('idleHandle');
        this.safeClearInterval('timeoutHandle');
        this.idling = true;
        this.running = false;
        this.timedOut = true;
        this.countdown = 0;
        this.onTimeout.emit(null);
    };
```

```js
   Idle.prototype.toggleState = function () {
        var _this = this;
        this.idling = !this.idling;
        if (this.timedOut) {
            this.toggleInterrupts(false);
        }
        if (this.idling) {
            this.onIdleStart.emit(null);
            this.stopKeepalive();
            if (this.timeoutVal > 0) {
                this.countdown = this.timeoutVal;
                this.doCountdown();
                this.timeoutHandle = setInterval(function () { _this.doCountdown(); }, 1000);
            }
        }
        else {
            this.timedOut = false;
            this.toggleInterrupts(true);
            this.onIdleEnd.emit(null);
            this.startKeepalive();
        }
        this.safeClearInterval('idleHandle');
    };
```

**Comment by HackedByChinese** [comment]
Fixed in d8d600d
